Data-Analytics/Spark-S3-secret/spark-secret-creation.ipynb

```diff
@@ -0,0 +1,85 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Spark S3 proxy secret\n",
```

Comment:
Maybe it's rather 'Spark S3 credentials secret'? It's not S3 proxy specific feature.